API/src/controllers/ordersController.js

```diff
@@ -0,0 +1,37 @@
+import Data from '../queries/orderQueries';
+
+class Order {
+  static getAllOrders(request, response) {
+    const data = Data.getAllOrders();
```

Comment:
Should a user be able to get Orders that are not his? Are there ways you can prevent that from happening here?

Reply:
@sudhons. I understand that, but don't you think you can filter out the Orders specific to a particular userId pending authentication?

Reply:
No users yet. no user, no userId.